src/custom/components/Header/index.tsx

```diff
@@ -0,0 +1,8 @@
+import HeaderMod from './HeaderMod'
+import styled from 'styled-components'
+
+export const Header = styled(HeaderMod)`
+  // CSS Override
+`
+
+export default Header
```

Comment:
exporting twice?

Reply:
yes, one default and one named. Will merge like that, but we can discuss if it's best not to do it like that